### ISSUE TITLE
Typo in brl-tutorial variable causing tutorial to fail

### ISSUE THREAD

**Issue by NICHOLAS85** [bedrocklinux/bedrocklinux-userland#149]
https://github.com/bedrocklinux/bedrocklinux-userland/blob/1b74405f7fb93213acd18d81a0c9b675cb3b3667/src/slash-bedrock/libexec/brl-tutorial#L360

```
/bedrock/libexec/brl-tutorial: line 349: strautm: parameter not set
```
${strautm} --> ${stratum}

**Comment by paradigm** [comment]
No idea how that slipped by me >.<

Thanks!  I'll close this once your fix is released.

**Comment by paradigm** [comment]
Should be fixed in the just released 0.7.11.  Thanks again!
